prow/plank/controller.go

```diff
@@ -412,14 +412,15 @@ func guberURL(pj kube.ProwJob, build string) string {
 	var url string
 	if pj.Spec.Type == kube.PresubmitJob || pj.Spec.Type == kube.BatchJob {
 		url = guberBasePR
+		if pj.Spec.Refs.Org != "kubernetes" {
```

Comment:
org is reasonable, but it does not tie to a repo though (can be none or multiple)